Fix failure to find Foundation.framework on macOS 11 and later.
https://github.com/IoLanguage/io/pull/381
https://github.com/IoLanguage/io/commit/6f6b750799e4c9c6e81acdc1bfd47e08036a8b47
--- addons/ObjcBridge/CMakeLists.txt.orig
+++ addons/ObjcBridge/CMakeLists.txt
@@ -40,9 +40,11 @@ set(SRCS
 set(CMAKE_EXE_LINKER_FLAGS "-framework AppKit")
 
 # Now build the shared library
+find_library(FOUNDATION Foundation)
+find_library(APPKIT AppKit)
 add_library(IoObjcBridge SHARED ${SRCS})
-add_dependencies(IoObjcBridge iovmall Foundation IoMD5 IoYajl IoBox IoSocket IoBlowfish IoSystemCall)
-target_link_libraries(IoObjcBridge iovmall ${ObjcBridge_LIBRARY} IoMD5 IoYajl IoBox IoSocket IoBlowfish IoSystemCall /System/Library/Frameworks/Foundation.framework/Versions/C/Foundation /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit)
+add_dependencies(IoObjcBridge iovmall IoMD5 IoYajl IoBox IoSocket IoBlowfish IoSystemCall)
+target_link_libraries(IoObjcBridge iovmall ${ObjcBridge_LIBRARY} IoMD5 IoYajl IoBox IoSocket IoBlowfish IoSystemCall ${FOUNDATION} ${APPKIT})
 
 # Install the addon to our global addons hierarchy.
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} DESTINATION lib/io/addons)
